modules/integration/tests-ui-integration/ui-test-utils/src/main/java/org/wso2/ds/ui/integration/util/DSUIIntegrationTest.java

```diff
@@ -134,11 +138,11 @@ public void popWindow() throws MalformedURLException, XPathExpressionException {
      * @param pwd      password
      * @throws javax.xml.xpath.XPathExpressionException,InterruptedException
      */
-    public void login(String userName, String pwd)
-            throws Exception {
+    public void login(String userName, String pwd) throws Exception {
         String fullUrl = "";
```

Comment:
Unnecessary assignment, could have merged this line with next line, and directly initialize with getBaseUrl() + DS_SUFFIX.